client/src/document/index.tsx

```diff
@@ -20,6 +20,7 @@ import { BrowserCompatibilityTable } from "./ingredients/browser-compatibility-t
 import { Breadcrumbs } from "../ui/molecules/breadcrumbs";
 import { Titlebar } from "../ui/molecules/titlebar";
 import { TOC } from "../ui/molecules/toc";
+import { RenderSideBar } from "./organisms/sidebar";
```

Comment:
I like that you didn't use `ingredients` because that's more about stuff *inside* the document. 
But what's the point of the `organisms` namespace? I'm not criticizing, I'm trying to understand. 
Not that "ingredients" is immediately obvious within documents, in a post-Stumptown era. 
It just doesn't feel that natural of a word. Perhaps "widgets" or "sub-components"?

Reply:
I guess the main reason is to keep the atomic design idea throughout instead off adopting something else/new here.